### PR TITLE
DAOS-2240 Add Jenkins RPM project repos to provisioned nodes

### PR DESCRIPTION
**Pull request by brianjmurrell**

### CODE REVIEW

File: vars/provisionNodes.groovy [hunk 1]
```diff
@@ -113,14 +119,24 @@ def call(Map config = [:]) {
                               chmod 600 /localhome/jenkins/.ssh/{authorized_keys,id_rsa*}
                               chown -R jenkins.jenkins /localhome/jenkins/.ssh
                               echo \\"jenkins ALL=(ALL) NOPASSWD: ALL\\" > /etc/sudoers.d/jenkins
+                              yum -y install yum-utils
+                              pkgs=\\"${inst_rpms}\\"
```
Comment (ashleypittman) [inline vars/provisionNodes.groovy]: Why use a local variable here, rather than just using inst_rpms on L139?

Comment (brianjmurrell) [reply vars/provisionNodes.groovy]: `$pkgs` is used on more than just L139. Yes, I could interpolate `inst_rpms` through all the places where `$pkgs` is used but it seems cleaner (and easier to lift this script to somewhere else if needed) to have a single interface point between the script and it's caller.

Comment (ashleypittman) [reply vars/provisionNodes.groovy]: Ah, I only saw the one instance.

**Comment by ashleypittman** [comment]
Does this still apply?  I can approve but it looks like it's probably been superseded now.

**Comment by brianjmurrell** [comment]
@ashleypittman This will be needed when we migrate to testing using the built RPMs but since it already has a conflict that needs resolving before it can be landed, let's just put it on the back-burner until we are ready to move to testing using RPMs.

**Pull request merged**